### ISSUE TITLE
stdin not supported

### ISSUE THREAD

**Issue by jayvdb** [smarkets/flake8-strict#3]
flake8 has a stdin mode, which isnt supported.

```
c:\python27\lib\site-packages\pep8-1.7.0-py2.7.egg\pep8.py:1521: in check_ast 
    for lineno, offset, text, check in checker.run(): 
c:\python27\lib\site-packages\flake8_strict.py:39: in run 
    errors = _process_file(self._filename) 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

filename = 'stdin' 

    def _process_file(filename): 
>       with open(filename, 'rt') as f: 
E       IOError: [Errno 2] No such file or directory: 'stdin'
```

**Comment by jstasiak** [comment]
@jayvdb This is fixed in master now (https://github.com/smarkets/flake8-strict/commit/3dbeac50e056ef11f11a1ce2671ff89cf61c43cb, pull request #2), will be released shortly.

**Comment by jstasiak** [comment]
So - "shortly" took longer than expected but the fix for this has just been released on PyPI in version 0.1.3.
